app/models/physical_server_provision_task.rb

```diff
@@ -0,0 +1,25 @@
+class PhysicalServerProvisionTask < MiqProvisionTask
+  alias_attribute :physical_server, :source
```

Comment:
We should avoid creating alias unless they are necessary.  Other task object use an `alias` because the once were separate tables before we merged them all into `miq_request_tasks`, and we wanted to keep them backwards compatible.  No need to do this for new models, we should reference the object as `source`.